qmv.sh

```diff
@@ -36,8 +36,8 @@ echo Compile Sakai from $mywd to $tomcatdir
 cd "$mywd/trunk"
 
 if command -v mvnd >/dev/null 2>&1; then
-    echo mvnd -e -Dmaven.test.skip=true -Dmaven.tomcat.home=/Users/csev/sakai/scripts/apache-tomcat-9.0.21/ -Dsakai.cleanup=true clean install sakai:deploy
-    mvnd -e -Dmaven.test.skip=true -Dmaven.tomcat.home=/Users/csev/sakai/scripts/apache-tomcat-9.0.21/ -Dsakai.cleanup=true clean install sakai:deploy
```

Comment:
I don't use mvnd, but that didn't look right to me. so Not tested.

Reply:
Why did you delete it? I use mvnd all the time.

Reply:
Oh - I did not see what you replaced it with :)  I will approve, merge, and check